dist/releases/embeds.go

```diff
@@ -40,3 +47,43 @@ func GetConfigFor(version string) ([]byte, error) {
 
 	return nil, fmt.Errorf("embedded config for version %s is not available; use one of %+v", version, GetVersions())
 }
+
+// GetJavaFile returns the java source file.
+func GetJavaFile() (fileFullPath, fileName string, err error) {
+	bytes, err := configs.ReadFile("java/" + JavaFips)
+	if err != nil {
+		return "", "", err
+	}
+
+	return createTempFile(bytes, "fipsJava")
+}
+
+// GetAlgorithmFile returns a file with disabled algorithms to check.
+func GetAlgorithmFile(javaDisabledAlgorithms []string) (fileFullPath, fileName string, err error) {
+	r := bytes.NewBuffer([]byte{})
+	for i := range javaDisabledAlgorithms {
+		r.WriteString(javaDisabledAlgorithms[i] + "\n")
+	}
+
+	return createTempFile(r.Bytes(), "disabledAlgorithms")
```

Comment:
I think you can replace this with something like
```go
return createTempFile([]byte(strings.Join(javaDisabledAlgorithms, "\n")), "disabledAlgorithms")
```

(with an intermediate variable, if you prefer it that way)

Reply:
good catch